### ISSUE TITLE
Implement first version of Flex component

### ISSUE THREAD

**Issue by trybern** [FHIDev/Fhi.Designsystem#212]
![Image](https://github.com/user-attachments/assets/d7e4b5af-a38b-476b-aac6-5583e1454d06)

ref: https://github.com/orgs/FHIDev/projects/4/views/1?pane=issue&itemId=112050102&issue=FHIDev%7CFhi.Designsystem%7C203

### Acceptance criteria

- It should be implemented in accordance [with the designs in Figma](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=9345-5179&p=f&t=vU8awaNokLqvOXIl-11)
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs (e.g form interactions)
- It should have appropriate component level design tokens that references semantic, and if appropriate, primitive level tokens.
